egs2/chime7_task1/asr1/local/data.sh

```diff
@@ -37,8 +37,8 @@ if [ ${stage} -le 0 ] && ! [[ " ${skip_stages[*]} " =~ " 0 " ]]; then
   cv_kaldi_manifests_ihm=()
   dset_part=dev
   mic=ihm
-  for dset in chime6 dipco; do
-    lhotse kaldi export -p ${manifests_root}/${dset}/${dset_part}/${dset}-${mic}_recordings_${dset_part}.jsonl.gz  ${manifests_root}/${dset}/${dset_part}/${dset}-${mic}_supervisions_${dset_part}.jsonl.gz data/kaldi/${dset}/${dset_part}/${mic}
+  for dset in chime6 dipco mixer6; do
+    lhotse kaldi export ${manifests_root}/${dset}/${dset_part}/${dset}-${mic}_recordings_${dset_part}.jsonl.gz  ${manifests_root}/${dset}/${dset_part}/${dset}-${mic}_supervisions_${dset_part}.jsonl.gz data/kaldi/${dset}/${dset_part}/${mic}
```

Comment:
I had to add the `--prefix-spk-id` flag in `lhotse kaldi export` otherwise the created data dir was not sorted by spk id.